mivs/templates/mivs_judging/index.html

```diff
@@ -16,7 +16,7 @@
 <tbody>
 {% for game in games %}
     <tr>
```

Comment:
I used the full URL since the purpose of that link is to allow an admin to right-click and select "Copy link address" and then get a full URL that they could copy into an email to the studio.

I noticed that Google Chrome will copy the full URL even if you select "Copy link address" on a relative link, but I wasn't sure if this was also the behavior of other browsers.  (I don't even have an Internet Explorer to test on.)

If it is the case that relative links will copy into fully qualified links then we can definitely remove the ``c.URL_BASE`` from this page.